content/sensu-go/5.9/getting-started/get-started.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Get started with Sensu"
 linkTitle: "Get Started with Sensu"
-description: "It’s easy to get started with Sensu Go. Download the sandbox, install our flexible monitoring software, and try our enterprise features. Get started with Sensu today."
+description: "Sensu is the industry leading solution for multi-cloud monitoring at scale. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflows and gain deep visibility into their multi-cloud environments. Get started now and feel the #monitoringlove."
```

Comment:
This is good, but in places where we've previously said (roughly): "get started sensu (classic) for free, then try our enterprise features", I think we can use some of the new terminology to encourage the same free-to-paid journey. 

I'm purposely not going to provide any quotes, because I don't actually think we need to change this now; this is just context for future copy: 
- The free version of Sensu Go is designed to help new and experienced users alike to get started and achieve quick time to value (or, to get immediate value); 
- the "license-activated" features in Sensu Go are there to help them "monitor at scale".

Reply:
+1 as discussed in my call with Alex, I think there's room for clarification under the free tier section on the docs page